blueprints/ember-frost-test/index.js

```diff
@@ -49,41 +52,46 @@ module.exports = {
    * package.json, so this is a temporary fix for that.
    */
   addPackagesToProject: function (packagesToAdd) {
```

Comment:
@job13er I did not change the `removeBowerPackagesFromProject` because I was waiting to get your feedback on this method before.